source/documentation/data-docs/data-faqs/index.md

```diff
@@ -32,18 +44,22 @@ Databases on the AP use Amazon Athena which allow you to query data using SQL. Y
 
 **The Amazon Athena workbench**: If you [log into the AWS console](https://aws.services.analytical-platform.service.justice.gov.uk) and click Services -> Athena, you'll see the Athena workbench. This is good for testing your queries.
 
+**Create a Derived Table**: You can also use [Create a Derived Table](https://user-guidance.analytical-platform.service.justice.gov.uk/tools/create-a-derived-table/index.html) to run SQL statements to query Athena.
+
 If you get `assumed-role/... is not authorized to perform: glue:GetDatabases`, [request database access](https://github.com/moj-analytical-services/data-engineering-database-access#standard-database-access).
 
-**Python/JupyterLab**: To run queries and/or read data into a pandas DataFrame, use `pydbtools`. More details are [here](https://github.com/moj-analytical-services/pydbtools). Remember to install the latest version!
+**Python**: To run queries and/or read data into a pandas DataFrame, use `pydbtools`. More details are [here](https://github.com/moj-analytical-services/pydbtools). Remember to install the latest version!
 
-**R/RStudio**: There is currently no single recommended package for querying databases in R. There is [dbtools](https://github.com/moj-analytical-services/dbtools) which should work on the "old" platform. [Rdbtools](https://github.com/moj-analytical-services/Rdbtools) should work on the "new" platform, but is not officially supported so should be used with caution.
+**R**: There is currently no single recommended package for querying databases in R. There is [dbtools](https://github.com/moj-analytical-services/dbtools) which should work on the "old" platform. [Rdbtools](https://github.com/moj-analytical-services/Rdbtools) should work on the "new" platform, but is not officially supported so should be used with caution.
```

Comment:
```suggestion
**R**: There is currently no single recommended package for querying databases in R. There is [dbtools](https://github.com/moj-analytical-services/dbtools) which should work on the "old" platform. [Rdbtools](https://github.com/moj-analytical-services/Rdbtools) should work on the "new" platform. This package is maintained by the analytical platform user community.
```

Reply:
Not sure what you mean by old and new platform?

Reply:
That text comes from the existing text on the guidance: https://user-guidance.analytical-platform.service.justice.gov.uk/data/data-faqs/#how-do-i-query-a-database-on-the-platform

I'll update to remove reference to 'old' and 'new'